completion/git-machete.completion.zsh

```diff
@@ -64,12 +64,12 @@ _git-machete() {
         (g|go)
           _arguments '1:: :__git_machete_directions_go' && ret=0
           ;;
-        (github)
-          _arguments '1:: :__git_machete_github_subcommands' && ret=0
-          ;;
         (help)
           _arguments '1:: :__git_machete_help_topics' && ret=0
           ;;
+        (github)
+          _arguments '1:: :__git_machete_github_subcommands' && ret=0
+          ;;
```

Comment:
Why  did you move it? It looks like the intent was to keep it in alphabetic order (it's a bit debatable whether it was or not, because 'github' is after 'g' but before 'go').

Reply:
Whoops but now it's after `help` 😅